### PR TITLE
create a pull request

### PR DESCRIPTION
**Pull request by Khaled-AlKhateeb**
YOUFLEX, capstone 🎥 :
This capstone project is developed by @inspecta and @Khaled-AlKhateeb.
Fetched shows from TV Maze API.
Displayed the shows dynamically using JavaScript.
Add likes to shows using involvement API.
Add popup window for the shows details.
Add comments section in the popup window that POST to the involvement API.
Fetch back the comments for each show and display them in the comment section.
Created 2 functions to count both likes and comments that are displayed.
Added JEST dependency.
Tested the count functions using JEST.

for more info about the project please see this [Zoom Video](https://drive.google.com/file/d/1CCYwOdMWnkQn97ZDO8wmf0f5MQBrVhFj/view?usp=sharing).

### CODE REVIEW

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hello @Khaled-AlKhateeb @inspecta   👋 ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

✖️ Invalid Code Review Request ✖️

You have submitted a project, that has failing linters. Kindly ensure that you fix the failing linter errors then request for a review in your dashboard.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Khaled-AlKhateeb  in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello TEAM! 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of GitHub Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] On Cards, Please show the number of comments as well as showing the number of likes 👍🏻 
Screenshot 2022-10-21 at 10 10 10 PM


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/comments-counter.js [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+  const commentBtn = el.querySelector('.comments-btn');
+  commentBtn.innerHTML = `Comment ${response.length}`;
+};
 
-const commentsCounter = (element) => {
-  commentsDisplay.innerHTML = `Comments (${element.children.length})`;
+const getCommentCounterApi = async (cArr) => {
+  for (let i = 0; i < cArr.length; i += 1) {
+    const commentUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/CleVZYCORgbN3niQTjHc/comments?item_id=${cArr[i]}`;
+    // eslint-disable-next-line no-await-in-loop
```
Comment (christianbiring1) [inline src/modules/comments-counter.js:10]: [OPTIONAL] 
-[ ] Kindly make sure you are not disabling some rules for the linters make sure your code keeps the standards

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Khaled-AlKhateeb @inspecta , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# REQUEST CHANGES :recycle: 
soclose

 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR summary
- [ ] Good commit message

 ## Required Changes ♻️
- [ ] As said by the previous review, Kindly make sure When a user clicks on the Like button of an item, the interaction should get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), and the screen gets updated. no need to refresh the page to get the actual number of `likes`
- [ ] The same applies to the comments on the popup.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi Team, 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by inspecta** [comment]
Thanks @sumairq

**Pull request merged**